packages/dev/core/src/Physics/v2/physicsBody.ts

```diff
@@ -338,14 +375,18 @@ export class PhysicsBody {
      * Filtering by body is inefficient. It's more preferable to register a collision callback for the entire world
      * and do the filtering on the user side.
      */
-    public registerOnCollide(func: (collider: PhysicsBody, collidedAgainst: PhysicsBody, point: Nullable<Vector3>) => void): void {
+    public registerOnCollide(
+        func: (collider: PhysicsBody, collidedAgainst: PhysicsBody, point: Nullable<Vector3>, distance: number, impulse: number, normal: Nullable<Vector3>) => void
+    ): void {
         return this._physicsPlugin.registerOnBodyCollide(this, func);
     }
 
     /**
      * Unregister a collision callback that is called when the body collides
      */
-    public unregisterOnCollide(func: (collider: PhysicsBody, collidedAgainst: PhysicsBody, point: Nullable<Vector3>) => void): void {
+    public unregisterOnCollide(

```

Comment:
Doc is missing for the param as well

Reply:
this code has been discussed in this PR. see previous comments.